Jenkinsfile

```diff
@@ -44,7 +44,9 @@ def smokeTest(gitUtils, bazel, utils) {
   goBuildNode(gitUtils, 'istio.io/istio') {
     bazel.updateBazelRc()
     utils.initTestingCluster()
-    def e2eArgs = "--logs_bucket_path ${gitUtils.logsPath()} "
+    def logHost = 'stackdriver'
+    def projID = 'istio-testing'
+    def e2eArgs = "--logs_bucket_path ${gitUtils.logsPath()} --log_provider=${logHost} --project_id=${projID} "
```

Comment:
@sebastienvas Is this the right way to set flag value? Even when I hardcoded string like this, the latest Jenkins run ([console output](https://testing.istio.io/job/istio/job/presubmit/777/console)) still seems to fail to pick these value up, since I print these values out in framework.go. Any suspection on what could be wrong?